src/custom-types.json

```diff
@@ -142,7 +142,12 @@
     ]
   },
   "application/tar": {
-    "compressible": true
+    "compressible": true,
+    "extensions": ["tar","tar.gz","taz","tgz"],
+    "sources": [
+      "https://www.gnu.org/software/tar/",
+      "https://en.wikipedia.org/wiki/Tar_(computing)"
```

Comment:
Links to wikipedia are not accepted as per the readme, you can remove this.

Reply:
Done. Thank you!